pkg/acsengine/defaults_test.go

```diff
@@ -503,6 +503,71 @@ func TestAgentPoolProfile(t *testing.T) {
 	}
 }
 
+// TestSetComponentsNetworkDefaults covers tests for setMasterNetworkDefaults and setAgentNetworkDefaults
+// TODO: Currently this test covers only api.Distro setting. Extend test cases to cover network configuration too.
+func TestSetComponentsNetworkDefaults(t *testing.T) {
+
+	var tests = []struct {
+		name string
+		//orchestrator to be tested
```

Comment:
pls put each comment at the end of the preceding line

Reply:
`//orchestrator to be tested` really this is just a friendly test name, right?

Reply:
yes